### PR TITLE
feat: [BT-1750]: Integrate tools in baseline image

### PR DESCRIPTION
**Pull request by AnshuH**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/37532)

### CODE REVIEW

**Comment by AnshuH** [comment]
trigger messagecheck authorcheck

**Comment by AnshuH** [comment]
trigger authorcheck

**Comment by AnshuH** [comment]
trigger authorcheck

**Comment by AnshuH** [comment]
trigger authorcheck

File: dockerization/base-images/Dockerfile-ubi-java-base [hunk 1]
```diff
@@ -18,6 +18,25 @@ RUN curl -OL https://github.com/adoptium/temurin11-binaries/releases/download/jd
...
+
+USER root
+
+RUN microdnf install -y yum wget \
+    && yum install dnf  -y \
+    && dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
+#    && dnf -y module disable postgresql \
+    && dnf install -y postgresql14 \
+    && dnf install -y postgresql14-server \
+    && wget https://repo.mongodb.org/yum/redhat/8/mongodb-org/1.0/x86_64/RPMS/mongodb-cli--1.0.0-rc1%7blatest_rpm%7d.x86_64.rpm -O mongo.rpm \
```
Comment (pranay-harness) [inline dockerization/base-images/Dockerfile-ubi-java-base:30]: This is bringing in 43 critical vulnerabilities from GO

**Comment by AnshuH** [comment]
trigger authorcheck

**Pull request closed without merging**